components/perl/Mail-SPF/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# This file was automatically generated using the following command:
+#   $WS_TOOLS/perl-integrate-module Mail::SPF
+#
+
+BUILD_STYLE = modulebuild
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_PERL_MODULE =		Mail::SPF
+HUMAN_VERSION =			2.9.0
+COMPONENT_REVISION =		6
+COMPONENT_SUMMARY =		Mail::SPF - An object-oriented implementation of Sender Policy Framework
```

Comment:
As we don't have any release of this package yet the COMPONENT_REVISION should be 0 or unset.

Reply:
Sorry overseen. Fixed now.